### PR TITLE
Add small code sample to custom checker documentation

### PR DESCRIPTION
**Pull request by epenet**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :scroll: Docs          |

## Description

Add small code sample to custom checker documentation

### CODE REVIEW

**Comment by epenet** [comment]
cc @cdce8p

File: doc/development_guide/how_tos/custom_checkers.rst [hunk 1]
```diff
@@ -113,7 +113,13 @@ It has the following format::
...
   This is used on the command line and in config files.
   The hyphen is replaced by an underscore when used in the checker,
-  similarly to how you would use  ``argparse.Namespace``.
+  similarly to how you would use  ``argparse.Namespace``:
+
+  .. code-block:: python
+
+    if self.linter.config.ignore_ints:
+        # skip check
+        pass
```
Comment (Pierre-Sassoulas) [inline doc/development_guide/how_tos/custom_checkers.rst:122]:
```suggestion
        return

**Review by Pierre-Sassoulas — APPROVED** [review]

File: doc/development_guide/how_tos/custom_checkers.rst (hunk 1 above)
Comment (epenet) [reply doc/development_guide/how_tos/custom_checkers.rst:122]: That was my original proposal, but `rstcheck` failed in CI:
`doc/development_guide/how_tos/custom_checkers.rst:122: (ERROR/3) (python) 'return' outside function`

I didn't want to make the code sample too big

Comment (Pierre-Sassoulas) [reply doc/development_guide/how_tos/custom_checkers.rst:122]: Maybe inverting the conditional and using an ellipsis instead of the pass then?

Comment (epenet) [reply doc/development_guide/how_tos/custom_checkers.rst]: Yep, that works

**Review by Pierre-Sassoulas — APPROVED** [review]
Thank you for making the doc better and congratulation on becoming a pylint contributor :)

**Pull request merged**